src/utils.ts

```diff
@@ -10,16 +10,18 @@ import { isNullOrUndefined } from 'util';
 
 class Utils {
 
-
 	private static _getPlatformConfig(platformId?: string): any {
 		try {
-			let config: {} = platformId ? require(`../platforms/${platformId}/config.json`) : {};
-			config = defaults(
-				config,
+			let config:{} = platformId ? require(`../platforms/${platformId}/config.json`) : {};
+			// this is a workaround, otherwise the config object is cached 
```

Comment:
loading from filesystem should be cached

Reply:
if its cached it wasn't working anymore, @maxklenk should check